### ISSUE TITLE
MicroBarChart: Accesibility properties not available hence not focusable by Tab key

### ISSUE THREAD

**Issue by sparkypro** [SAP/ui5-webcomponents-react#3265]
**Describe the bug**
MicroBarChart: Accesibility properties not available hence not focusable by Tab key

**UI Link**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/charts-microbarchart--default-story

**To Reproduce**

1. Go to UI Link
2. Use Tab key from keyboard to focus on the MicroBarChart UI.

**Expected behavior**
MicroBarChart ui must be reachable by Tab key from keyboard, and must have accessibility properties to read out child elements

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.5.0
`@ui5/webcomponents-react` version: 0.26.1
Operating System: Windows 11
Browser: Chrome

**Comment by Lukas742** [comment]
The MicroBarChart was a remnant of outdated specifications. Currently there is no concept for this component, therefore we removed it with our v2 release. Updating the component in v1 is not planned and therefore I'm going to close this issue.
